proxy/upstream_test.go

```diff
@@ -0,0 +1,69 @@
+package proxy
+
+import (
+	"context"
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/coinbase/redisbetween/config"
+	"github.com/coinbase/redisbetween/utils"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
```

Comment:
Nit: Pull down `github.com` and `go.uber.org` imports to separate group

Reply:
done